README.md

```diff
@@ -14,6 +14,17 @@ to alert data, therefore safe to be accessed by wider audience.
 Alertmanager's API isn't stable yet and can change between releases.
 unsee currently supports Alertmanager `0.4` and `0.5`.
 
+## Security
+
+unsee process doesn't send any API request to the Alertmanager that could modify
```

Comment:
I'd suggest a small amount or rewording here. Either "Unsee doesn't" or "The Unsee process doesn't". Same goes for the first bullet point. Then either "to Alertmanager" or "to the Alertmanager API".